global_dist_boundary: 0.3
global_angle_boundary: 0.05 #0.05
HJ_MODE: 2  #0 Hanjeon, 1 docking postech, 2 regular move postech
Without_QR_move: true # HJ_MODE 2 + true for only driving without QR intialization
Main_start_x: 0.311524927616
Main_start_y: -0.00846561
Main_goal_x: 2.33031272888
Main_goal_y: -3.06516885757e-05
